### ISSUE TITLE
Doesn't commend anymore like before why?

### ISSUE THREAD

**Issue by dStroke** [Marc3842h/Titan#97]
It can commend only 1 time 1 account only? 
It doesn't work anymore correctly, it says it send a commends but the accounts doesn't receive them
and It bugs so much times that I should login on every account to disconnect them from playing CS:GO
It doesn't disconnect accounts correctly

**Comment by rishav394** [comment]
As far as ingame mechanics are concerned you can commend a player only once from one steam ID. Despite the confirmation that the target was commended no additional commends are added if you have already commended the target from that very ID.

**Comment by Marc3842h** [comment]
Hi @dStroke,
As @rishav394 has already pointed out, commending is only possible once per bot to a target. After that the bot can not commend the target again (this is a limitation from CS:GO itself). If you try to do it regardless, you'll never get a response of a successful confirmation and your account won't disconnect until it timed out (60 seconds for unprotected and 2 minutes for protected accounts).

Feel free to reply if you have more questions. If you find a actual bug, please another another issue as I flagged this issue as a `question`.
